data/35e/wizards_of_the_coast/supplement/complete_adventurer/cad_skills.lst

```diff
@@ -1,3 +1,3 @@
 
 
-Craft (Poisonmaking)										KEYSTAT:INT	USEUNTRAINED:NO						TYPE:Intelligence.Craft.Standard.Base.Craft (Poisonmaking)													SOURCEPAGE:p.97
+Craft (Poisonmaking)				KEYSTAT:INT	USEUNTRAINED:NO				TYPE:Intelligence.Craft.Standard.Base	VISIBLE:DISPLAY			SOURCEPAGE:p.97
```

Comment:
Mate,

I do not understand the intent of this change.

VISIBLE:DISPLAY equates to GUI only. 
Removing the TYPE does not make any sense.

What is this supposed to be fixing?